pvlib/test/test_pvsystem.py

```diff
@@ -577,6 +578,16 @@ def test_snlinverter(sam_data):
     pacs = pvsystem.snlinverter(vdcs, pdcs, inverters[testinv])
     assert_series_equal(pacs, pd.Series([-0.020000, 132.004308, 250.000000]))
 
+def test_adrinverter(sam_data):
+    inverters = sam_data['adrinverter']
+    testinv = 'Zigor__Sunzet_3_TL_US_240V__CEC_2011_'
+    vdcs = pd.Series(np.linspace(0,50,3))
+    idcs = pd.Series(np.linspace(0,11,3))
+    pdcs = idcs * vdcs
+
+    pacs = pvsystem.adrinverter(vdcs, pdcs, inverters[testinv])
+    assert_series_equal(pacs, pd.Series([-0.25000, 105.635043, 503.769061]))
+
```

Comment:
This test may catch some types of future bugs, but it's not a good PV example.  The current, voltage and power ranges are way too small for this Zigor model inverter.  

Another idea might be to use some voltage and current combinations found in the pdf file on the CEC/gosolar website and look for the results of the model to be approxiamtely equal.